roles/network_plugin/cilium/templates/hubble/deploy.yml.j2

```diff
@@ -61,6 +61,7 @@ spec:
             name: tls
             readOnly: true
           {% endif %}
+
```

Comment:
HI @toonalbers 

Thanks for the fix :-)

It seems an issue about the [whitespace-control of jinjia](https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control).

It can be solve by the code:
```
          {%- if cilium_hubble_tls_generate %}
          - mountPath: /var/lib/hubble-relay/tls
            name: tls
            readOnly: true
         {%- endif %}

```
The same usage is common in the kubespray code :-)
we can check the template in the https://j2live.ttl255.com/

Reply:
Hi @yankay,

Thank you for the review! I agree that adding the minus does result in neater YAML, but the newline is also still required. On https://j2live.ttl255.com/ it works fine without a newline, but in practice (at least with the Ansible version from requirements.txt) the result is malformed YAML.

Minus only:
```sh
$ kubectl apply -f /etc/kubernetes/addons/hubble/hubble-deploy.yml --dry-run=server
error: error parsing /etc/kubernetes/addons/hubble/hubble-deploy.yml: error converting YAML to JSON: yaml: line 61: mapping values are not allowed in this context
```

With the offending line 61:
```yaml
56           - mountPath: /etc/hubble-relay
57             name: config
58             readOnly: true
59           - mountPath: /var/lib/hubble-relay/tls
60             name: tls
61             readOnly: true      restartPolicy: Always
62       serviceAccount: hubble-relay
63       serviceAccountName: hubble-relay
```

Newline only:
```yaml
56           - mountPath: /etc/hubble-relay
57             name: config
58             readOnly: true
59           - mountPath: /var/lib/hubble-relay/tls
60             name: tls
61             readOnly: true
62
63       restartPolicy: Always
64       serviceAccount: hubble-relay
65       serviceAccountName: hubble-relay
```

Newline and minus:
```yaml
54             name: hubble-sock-dir
55             readOnly: true
56           - mountPath: /etc/hubble-relay
57             name: config
58             readOnly: true
59           - mountPath: /var/lib/hubble-relay/tls
60             name: tls
61             readOnly: true
62       restartPolicy: Always
63       serviceAccount: hubble-relay
64       serviceAccountName: hubble-relay
```

I will update the PR to add the minus to the front of the tags.